acceptance-tests/src/acceptance-test/java/tech/pegasys/artemis/test/acceptance/SyncAcceptanceTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.test.acceptance;
+
+import java.util.function.Consumer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.artemis.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.artemis.test.acceptance.dsl.ArtemisNode;
+import tech.pegasys.artemis.test.acceptance.dsl.ArtemisNode.Config;
+import tech.pegasys.artemis.test.acceptance.dsl.tools.GenesisStateConfig;
+
+public class SyncAcceptanceTest extends AcceptanceTestBase {
+  private static final Logger LOG = LogManager.getLogger();
+
+  @Test
+  @Disabled("This test doesn't work yet.  Need to debug node connectivity issues.")
+  public void shouldSyncToNodeWithGreaterFinalizedEpoch() throws Exception {
+    final int validatorCount = 2;
+    final GenesisStateConfig genesisStateConfig = GenesisStateConfig.create(validatorCount);
+
+    final ArtemisNode primaryNode =
+        createArtemisNode(configurePrimaryNode(genesisStateConfig, validatorCount));
+    final ArtemisNode lateJoiningNode =
+        createArtemisNode(configureLateJoiningNode(genesisStateConfig, primaryNode));
+
+    LOG.debug("Start first node");
+    primaryNode.start();
+    LOG.debug("Wait for genesis");
```

Comment:
These log statements are a good idea - we should put them in the DSL (ie `ArtemisNode.start()` should automatically print a debug log).  The messages would be slightly less nice (each node has a unique name but it's not as nice a "first node") but it would make the tests simpler and give better progress indicators for all tests.

Reply:
Good call 👍